src/components/sections/communities/_partials/Section.tsx

```diff
@@ -37,7 +37,7 @@ export default function Section({ title, titleBold = true, subtitle, id, hideSub
   return (
     <div id={id} className={`text-xl md:text-.5xl px-0 py-6 ${className}`}>
       {title && <H3 bold={titleBold}>{title}</H3>}
-      <p className={`text-base md:text-.5xl font-normal leading-normal ${hideSubtitleOnMobile ? "hidden" : ""}`}>{subtitle}</p>
+      <p className={`text-base md:text-.5xl font-normal leading-normal text-surface-text-primary ${hideSubtitleOnMobile ? "hidden" : ""}`}>{subtitle}</p>
```

Comment:
even here

Reply:
@musayann 
In our Figma design, we have duplicate color names within the Surface section. For instance, the "Secondary" color used for text is different from the "Secondary" color used for backgrounds. To avoid naming collisions, I need a more descriptive naming convention.
Instead of using generic names like `surface-tertiary` for both text and background colors, which would cause conflicts, I used a more specific naming scheme. 
That is why I used these namings, 
`surface-text-secondary` for the text color
`surface-bg-secondary` for the background color
which ensures clarity and prevents any confusion or overlap between different color usages.

https://www.figma.com/design/0BRR6Ch2qbOEUT4zIdmmzg/Dacade-2?node-id=2238-33014&t=Gn4xvKkhVNfk8Nus-4